app/views/portal/show.rb

```diff
@@ -412,12 +417,118 @@ def content
               }
             end
           },
+          named_entities: named_entity_data,
           hierarchy: @hierarchy.blank? ? nil : record_hierarchy(@hierarchy),
           thumbnail: render_document_show_field_value(document, 'europeanaAggregation.edmPreview', tag: false)
         }.reverse_merge(helpers.content)
       end
     end
 
+    def named_entity_data()
```

Comment:
Method has too many lines. [11/10]<br>Omit the parentheses in defs when the method doesn't accept any arguments.